packages/base/Notification/src/Notification/Notification.tsx

```diff
@@ -44,69 +48,59 @@ export interface PrivateProps
 /** `elevated` and `icon` props are omitted from the public declaration, since they're only for internal use */
 export type PublicProps = Omit<PrivateProps, 'elevated' | 'icon'>
 
-const renderNotificationCloseButton = ({
-  onClose,
-  classes,
-  testIds,
-}: PrivateProps) => (
+const renderNotificationCloseButton = ({ onClose, testIds }: PrivateProps) => (
   <ButtonCircular
     data-testid={testIds?.closeButton}
     onClick={onClose}
-    className={classes?.close}
-    icon={<CloseMinor16 className={classes?.closeIcon} />}
+    className='absolute right-[0.5em] top-[0.75em] bg-[transparent] border-[0] p-0 h-[1em] hover:bg-[transparent]'
+    icon={<CloseMinor16 className='fill-graphite-700' />}
   />
 )
 
-const renderNotificationIcon = ({ icon, variant, classes }: PrivateProps) => {
-  const iconProps = {
-    className: classes?.icon,
-  }
-
+const renderNotificationIcon = ({ icon, variant = 'yellow' }: PrivateProps) => {
   // TODO: these are Icons required circular Icon bg color definitions, all Icons should be white on that color
   // Missing the following: https://github.com/toptal/picasso/issues/253
   switch (variant) {
     case 'red':
-      return <ExclamationSolid24 {...iconProps} color='red' />
+      return <ExclamationSolid24 color='red' />
 
     case 'yellow':
-      return <ExclamationSolid16 {...iconProps} color='yellow' />
+      return <ExclamationSolid16 color='yellow' />
 
     case 'green':
-      return <CheckSolid24 {...iconProps} color='green' />
+      return <CheckSolid24 color='green' />
 
     default: {
-      const infoProps = { ...iconProps, color: 'grey' as const }
+      const infoProps = { color: 'grey' as const }
 
       return icon ? cloneElement(icon, infoProps) : <Info24 {...infoProps} />
     }
   }
 }
 
 const renderNotificationContent = (props: PrivateProps) => {
-  const { classes, children, onClose, variant, testIds } = props
-
-  const capitalizedVariant = capitalize(variant as string)
+  const { children, onClose, variant, testIds } = props
 
   return (
     <Container
       flex
-      className={classes?.contentWrapper}
+      className='flex w-full my-0 mx-auto p-0 max-w-[75em] min-w-[0]'
       data-testid={testIds?.content}
     >
       <Container
         flex
         alignItems='center'
-        className={cx(
-          classes?.iconWrapper,
-          classes?.[`iconWrapper${capitalizedVariant}`]
+        className={twMerge(
+          'h-[1.5em] min-w-[1.5em] mr-[1em] basis-[1.5em]',
+          variant === 'yellow' && 'h-[1em] min-w-[1em] mt-[2px] basis-0'
         )}
```

Comment:
```suggestion
        className={twJoin(
          'min-w-[1.5em] mr-[1em] basis-[1.5em]',
          variant === 'yellow' ? 'h-[1em] min-w-[1em] mt-[2px] basis-0' : 'h-[1.5em]'
        )}
```

Here we are using `twMerge` only for the `h-*` className, I think we can just have simple condition instead of using twMerge.

Reply:
twMerge is not fee, please read closely: https://github.com/dcastil/tailwind-merge/blob/v2.3.0/docs/when-and-how-to-use-it.md

Reply:
Yes, I'm aware that `twMerge` consumes resources and should only be used in specific cases, but I haven't seen the refactoring so that I can apply `twJoin`. I have updated the className, thanks both for your suggestions.